listenbrainz/webserver/static/js/src/LastFMImporter.test.tsx

```diff
@@ -172,11 +172,18 @@ describe("getPage", () => {
       });
     });
 
-    await instance.getPage(1);
-    // There is no direct way to check if retry has been called
-    expect(setTimeout).toHaveBeenCalledTimes(1);
+    const getPageSpy = jest.spyOn(instance, "getPage");
+    await instance.getPage(1, LASTFM_RETRIES);
 
-    jest.runAllTimers();
+    // we run the timer sufficient number of times to ensure retries are not exceeded or undetected due to timeouts
+    for (let i = 0; i < LASTFM_RETRIES + 2; i += 1) {
+      jest.runAllTimers();
+      // make timers and promises play well with jest
+      // eslint-disable-next-line no-await-in-loop
+      await Promise.resolve();
+    }
+
+    expect(getPageSpy).toHaveBeenCalledTimes(4);
```

Comment:
what's this 4? Is it LASTFM_RETRIES + 2 ? either add a comment explaining how it's computed, or use a constant

Reply:
`1 + LASTFM_RETRIES`, fixed.